src/concrete/ml/deployment/fhe_client_server.py

```diff
@@ -149,17 +175,22 @@ def __init__(self, path_dir: str, model: Any = None):
 
         Path(self.path_dir).mkdir(parents=True, exist_ok=True)
 
-    def _export_model_to_json(self) -> Path:
+    def _export_model_to_json(self, is_training: bool = False) -> Path:
         """Export the quantizers to a json file.
 
+        Args:
+            is_training (bool): If True, we export the training mode.
```

Comment:
`If True, we export the training mode.` -> `module` or `circuit`, but not `mode` right ?

Reply:
changed to circuit